v2/schema/context.jsonld

```diff
@@ -0,0 +1,112 @@
+{
+  "@context": {
+    "dcat": "http://www.w3.org/ns/dcat#",
+    "@vocab": "http://www.w3.org/ns/dcat#",
+    "dc": "http://purl.org/dc/terms/",
+    "dctypes": "http://purl.org/dc/dcmitype/",
```

Comment:
i think we need both `dc` (for 1.1 elements) and `dcterms` (for terms as ^^^) because things like title are typically referred to via the 1.1 element set. it's very confusing.